templates/node/lib/client.js.twig

```diff
@@ -11,6 +12,7 @@ class Client {
         this.headers = {
             'accept-encoding': '*',
             'content-type': '',
+            'user-agent' : `{{spec.title | caseUcfirst}}{{language.name | caseUcFirst}}SDK/{{ sdk.version }} (${os.type()} ${os.version()}; ${os.arch()})`,
```

Comment:
should the ; be after the os type rather than after the version?

Reply:
updated